orchagent/portsorch.cpp

```diff
@@ -461,6 +462,107 @@ void PortsOrch::updateDbPortOperStatus(sai_object_id_t id, sai_port_oper_status_
     }
 }
 
+bool PortsOrch::createPort(const set<int> &lane_set, uint32_t speed)
```

Comment:
change the name to `addPort`? it then aligns all the other add/remove functions in SwSS.